client/me/billing-history/index.jsx

```diff
@@ -69,12 +63,16 @@ module.exports = React.createClass( {
 					<BillingHistoryTable transactions={ data.billingHistory } />
 				</Card>
 
-				<SectionHeader label={ this.translate( 'Upcoming Charges' ) } />
-				<Card id="upcoming-charges">
-					<UpcomingChargesTable sites={ this.props.sites } transactions={ data.upcomingCharges } />
-				</Card>
+				{ hasBillingHistory &&
+					<div>
+						<SectionHeader label={ this.translate( 'Upcoming Charges' ) } />
+						<Card id="upcoming-charges">
+							<UpcomingChargesTable sites={ this.props.sites } transactions={ data.upcomingCharges } />
+						</Card>
+					</div> }
 
-				{ this.renderManageCards() }
+				{ config.isEnabled( 'me/credit-cards' ) &&
+					<CreditCards cards={ storedCards } /> }
```

Comment:
@ebinnion I originally suggested it as it currently is and that's how I've seen it most in the existing codebase. I was a bit uneasy at first too but I ended up liking how it didn't take up a bunch of room